jaxrs-spec/src/main/asciidoc/chapters/applications/_publication.adoc

```diff
@@ -15,9 +15,18 @@ Applications are published in different ways depending on whether the
 application is run in a Java SE environment or within a container. This
 section describes the alternate means of publication.
 
+A compliant implementation MUST support both alternatives on Java SE.
+
 [[java-se]]
 ==== Java SE
 
+There are two alternative ways of publication on Java SE:
+The full Java SE HTTP server bootstrap, and creating just Java SE Endpoints.
```

Comment:
This sentence does not read well. How about: "There are two alternative ways of publishing on Java SE: creating SE endpoints directly or using the SE bootstrap API."

Reply:
Yes this reads much better, but it does not explain what the SE bootstrap API *is*. What do others think?

Reply:
Your sentence does not explain that either, that's why you have a sub-section for that.

Reply:
I like @spericas's wording. And I think that the more detailed explanation in the subsection about what the SE bootstrap API actual does is fine.

Reply:
> Your sentence does not explain that either, that's why you have a sub-section for that.

Well in fact my sentence does not need to, as it clearly says what the differences are: Either you get an HTTP server, or you get an endpoint.

Anyways, I have no problem with chaning this sentence.

Reply:
Proposed sentence is included now in commit https://github.com/eclipse-ee4j/jaxrs-api/pull/932/commits/1ffdd0877985ca2ce92fb0eadde9aef86cb84026.